doc/nrf/includes/sample_fem_support.txt

```diff
@@ -4,6 +4,9 @@ FEM support
 .. note::
    Software FEM support is not present on nRF53 Series devices yet.
 
+.. note::
+   Software FEM support can't be combined with multiprotocol operation yet.
+
```

Comment:
Is this valid for Thread samples as well?
Can the two notes be combined into one?